src/components/functionselect/functionselect.js

```diff
@@ -14,25 +14,50 @@ angular.module('vlui')
 
         scope.func = {
           selected: undefined,
-          list: [undefined]
+          list: {
+            aboveFold: [],
+            belowFold: [] // could be empty
+          }
         };
 
-        function getFns(type) {
+        // timeUnits for T
+        var timeUnits = {
+          aboveFold: ["year", "quarter", "month", "date", "day", "hours", "yearmonthdate"],
+          belowFold: ["hoursminutes", "hoursminutesseconds", "milliseconds", 
```

Comment:
I noticed that vega-lite doesn't have `yearmonthdatehours`, `yearmonthdatehoursminutes`, `yearmonthdatehoursminutesseconds`, but vega-lite has `yearmonthdayhours` etc.

To me, `2016-July-15-13:50:00` (`yearmonthdatehousminutesseconds`) makes more sense than `2016-July-Friday-13:50:00` (`yearmonthdayhoursminutesseconds`).